customcommands/assets/customcommands-database.html

```diff
@@ -20,6 +20,25 @@ <h2>Custom Commands Database</h2>
 </header>
 {{template "cp_alerts" .}}
 
+<div>
+    <section class="card">
+        <header class="card-header">
+            <h2 class="card-title">Database Usage - {{.DatabaseUsagePercent}}% Full</h2>
+        </header>
+        <div class="card-body">
+            <div class="col-auto">
+                <h4>{{.TotalDatabaseUsage}} Entries used of total {{.TotalDatabaseCapacity}}</h4>
```

Comment:
Thoughts on this change, given that we use the serialised value's length to calculate the size of a value, rather than its in-memory size (primarily relevant for slices and maps)? Obviously miniscule nitpicking only for justifying me reviewing this.
```suggestion
                <h4>Approximately {{.TotalDatabaseUsage}} entries used of total {{.TotalDatabaseCapacity}}</h4>
```

Reply:
Sorry, could you clarify? From my understanding, the max database entries is calculated by count of entries alone, and not their serialized size. Therefore the important info to show would be how many entries are used and how many are remaining. Is that not accurately calculated with this PR?

Reply:
Oh oopsie daisy, I misunderstood you then. Please ignore this then